### DISCUSSION TITLE
A new feature in the visualiser - clipping

### DISCUSSION THREAD

**Discussion by sgenoud** [sgenoud/replicad#59] [Visualiser]
I finally found some time to finish this new visualiser functionality: clipping.

<img width="773" alt="Capture d’écran 2022-09-27 à 17 01 53" src="https://user-images.githubusercontent.com/263325/192564439-39589107-69c9-4b9f-9414-c97cd7c24c21.png">

I often found myself wanting to be sure that a tolerance is well implemented (and seen myself using other tools to do that). Now I can just use the clipping functionality.

As always, feedback welcome!

**Comment by tve** [comment]
Something Id love to see in the visualizer is to be able to easily toggle a bunch of items on/off (view/hide), where items would be shapes, clipping places, and markers (lines or points that mark stuff but don't get exported). This way instead of fiddling with entering a Y position for the clipping plane in your example one could return a bunch of clipping planes from main, whose position got calculated.
